spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -204,6 +204,21 @@
         expect(lan.reload.name).to eq("DC0_DVPG1_RENAMED")
       end
 
+      it "adding a new distributed virtual portgroup" do
```

Comment:
There wasn't an issue with newly created dvportgroups but figured it was safer to add a test for it anyway